code-challenges.test.js

```diff
@@ -13,49 +13,92 @@
-const letterE = "e"
-// Expected output: ["Cherry", "Blueberry", "Peach"]
+// describe(fruits, () => {
 
+//     const arrayOfWords1 = ["Apple", "Banana", "Plum", "Orange", "Kiwi"]
+//     const letterA = "a"
+//     const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
+//     const letterE = "e"
 
+//     it("Create a function that takes in an array of words and a single letter and returns all the words that contain that particular letter.", () => {
+//         expect(fruits(arrayOfWords1, letterA)).toEqual(["Apple", "Banana", "Orange"])
+//         expect(fruits(arrayOfWords2, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+//     })
+// })
 
-// b) Create the function that makes the test pass.
+Create a function that makes the test passes
+Uses filter to filter through the words that contains a particular letter
+Use Lowercase to ensure the letter is return into the word in lowercase.
 
+    // b) Create the function that makes the test pass.
 
+    // const stupidFunction = (wordsarray, letter) => {
+    //     return wordsarray.filter(word => word.toLowerCase().includes(letter))
+    // }
 
-// --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
-// a) Create a test with expect statements using the variable provided.
+    // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
-const hand1 = [5, 5, 5, 3, 3]
-// Expected output: true
-const hand2 = [5, 5, 3, 3, 4]
-// Expected output: false
-const hand3 = [5, 5, 5, 5, 4]
-// Expected output: false
+    // a) Create a test with expect statements using the variable provided.
 
+    describe(fullHouse, () => {
 
+        const hand1 = [5, 5, 5, 3, 3]
+        const hand2 = [5, 5, 3, 3, 4]
+        const hand3 = [5, 5, 5, 5, 4]
 
-// b) Create the function that makes the test pass.
+        it("Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house.", () => {
+            expect(fullHouse(hand1)).toEqual(true)
+            expect(fullHouse(hand2)).toEqual(false)
+            expect(fullHouse(hand3)).toEqual(false)
+        })
+    })
+
+//b) Create the function that makes the test pass.
+
+// Pseduo code: 
+// - it is a full house when exactly one pair and one three of a kind
+// - find a unique number that does not repeat, is not a full house
+// - use .set and .length
+// - return if Boolean vaule
+
+const fullHouse = (array) => {
+    let unique = 0
+    if ([...new Set(arrary)].length === 2 && array.length === 5) {
+
+    }
+}
+// not sure if I am on the right track with this function. I was trying to use the new Set() to create a new array, not sure what else I needed to do with this function.
```

Comment:
I can see your logic here. I think a problem may occur if the array had four of a kind.